pyvista/plotting/render_window_interactor.py

```diff
@@ -940,7 +1086,7 @@ def _mouse_right_button_click(self, x=None, y=None, count=1):
 
     def _mouse_move(self, x, y):  # pragma:
         """Simulate moving the mouse to ``(x, y)`` screen coordinates."""
-        self.interactor.SetEventInformation(x, y)
+        self.interactor.SetEventPosition(x, y)
```

Comment:
This change was because `SetEventInformation` resets all the controlling key presses, while `SetEventPosition` does not.

Reply:
Good catch!